src/calendars/philippines.ts

```diff
@@ -38,7 +39,18 @@ const dates = async (config: RomcalConfig): Promise<Array<LiturgicalDayInput>> =
     {
       key: 'pedro_calungsod_martyr',
       rank: Ranks.MEMORIAL,
-      date: dayjs.utc(`${year}-4-2`),
+      date: ((year: number): dayjs.Dayjs => {
+        const date = dayjs.utc(`${year}-4-2`);
+        const palmSunday = Dates.palmSunday(year);
+        const divineMercySunday = Dates.divineMercySunday(year);
+        // When 2 April is prevented, because of a Sunday of lent taking precedence
+        // or when it falls within Holy Week or the Octave of Easter,
+        // his feast is transferred to the Saturday before Palm Sunday
```

Comment:
```suggestion
        // When 2 April occurs with a Sunday of Lent or the Holy Week
        // or the Octave of Easter, the celebration is transferred
        // to Saturday before Palm Sunday.
```

Although I am not exactly sure if _preceding Saturday if date falls on a Sunday of Lent_ means _any_ Sunday of Lent or just Palm Sunday and Sunday of Divine Mercy.

Reply:
```suggestion
        // When 2 April occurs with a Sunday of Lent or the Holy Week
        // or the Octave of Easter, the celebration is transferred
        // to the Saturday before Palm Sunday.
```

Thank you for your change suggestions. In the last line, I'll keep the _"the" Saturday_, because it's a well-defined day. 

> Although I am not exactly sure if preceding Saturday if date falls on a Sunday of Lent means any Sunday of Lent or just Palm Sunday and Sunday of Divine Mercy.

By reading different sources, in my understanding, when this memorial is prevented, it is moved to the Saturday before Palm Sunday. We will refine this rule if we can get more (official) details.

Reply:
Okay, LGTM. 😉